rise-google-sheet.html

```diff
@@ -292,17 +292,15 @@
         this._startTimer();
       },
 
-      _handleNoNetwork: function( resp ) {
+      _handleErrorsWithCachedData: function( resp ) {
```

Comment:
@rodrigopavezi Okay, the name made me expect to see an opposite function *_handleErrorsWithoutCachedData* in the code also, but leave it as is, thanks.